index.js

```diff
@@ -545,6 +545,7 @@ function getBasicRequestOptions (method, defaultPath, headers, opts, agent) {
 function getHeaders (opts) {
   const headers = {}
   if (opts.secretToken) headers.Authorization = 'Bearer ' + opts.secretToken
+  if (opts.apiKey) headers.Authorization = 'ApiKey ' + opts.apiKey
   headers.Accept = 'application/json'
```

Comment:
I see here that when we're generating headers to send to the HTTP server that we consult the configuration for an `apiKey` value.  If this value is present we prefer its configuration over the secret token authorization as a value for the authorization header. This value is passed "raw" from the configuration with no extra encoding -- this means either the base64 encoding happens in the agent proper, or that we're expecting users to provide us with a base64 encoded key. 👍